pandas/tseries/tests/test_timeseries.py

```diff
@@ -1,42 +1,38 @@
 # pylint: disable-msg=E1101,W0612
 import calendar
-from datetime import datetime, time, timedelta
-import sys
 import operator
+import sys
 import warnings
+from datetime import datetime, time, timedelta
+from numpy.random import rand
+from numpy.testing.decorators import slow
+
 import nose
 import numpy as np
-import pandas.tseries.frequencies as frequencies
+import pandas.index as _index
 import pandas.lib as lib
 import pandas.tslib as tslib
-import pandas.index as _index
-import pandas as pd
-from pandas import (Index, Series, DataFrame, isnull, date_range, Timestamp,
-                    Period, DatetimeIndex, Int64Index, to_datetime,
-                    bdate_range, Float64Index, NaT, timedelta_range, Timedelta)
 
-from pandas.compat.numpy_compat import np_datetime64_compat
+import pandas as pd
+import pandas.compat as compat
+import pandas.core.common as com
 import pandas.core.datetools as datetools
+import pandas.tseries.frequencies as frequencies
 import pandas.tseries.offsets as offsets
 import pandas.tseries.tools as tools
-
-
-from pandas.util.testing import assert_series_equal, assert_almost_equal,\
-    _skip_if_has_locale
 import pandas.util.testing as tm
-
-from pandas.tslib import iNaT
-
+from pandas import (
+    Index, Series, DataFrame, isnull, date_range, Timestamp, Period,
+    DatetimeIndex, Int64Index, to_datetime, bdate_range, Float64Index,
+    NaT, timedelta_range, Timedelta, _np_version_under1p8, concat,
+    PeriodIndex)
 from pandas.compat import range, long, StringIO, lrange, lmap, zip, product
-from numpy.random import rand
-from pandas.util.testing import assert_frame_equal
+from pandas.compat.numpy_compat import np_datetime64_compat
 from pandas.core.common import PerformanceWarning
-import pandas.compat as compat
-import pandas.core.common as com
-from pandas import concat
-from pandas import _np_version_under1p8
-
-from numpy.testing.decorators import slow
+from pandas.tslib import iNaT
+from pandas.util.testing import (
+    assert_frame_equal, assert_series_equal, assert_almost_equal,
+    _skip_if_has_locale, assert_index_equal)
```

Comment:
its fine. problem is these things are cleaned differntly based on settings. 

I guess it reordered it a bit more logically, ok then.